src/plugins/data/server/search/es_search/get_default_search_params.ts

```diff
@@ -43,10 +43,15 @@ export async function getDefaultSearchParams(uiSettingsClient: IUiSettingsClient
   };
 }
 
+export interface AsyncOptions {
```

Comment:
In the same way I don't want to include `ignoreThrottled` in OSS, I'd prefer if we left these out of OSS code. Again, I understand it's not totally related to this PR.